index.html

```diff
@@ -977,13 +1003,50 @@
 				html = html + "<span class='varname'>Institute: </span> <span class='attribute'>" + data.institute + "</span>";
 				html = html + "<br />"
 				html = html + "<span class='varname'>Campus: </span> <span class='attribute'>" + standardizeCase(data.campus) + "</span>";
-				html = html + "<br />"
-				html = html + "<span class='varname'>School District: </span> <span class='attribute'>" + standardizeCase(data.district) + "</span>";
+				if (data.district) {
```

Comment:
I noticed that some RSL points (e.g. Comal ISD on the E side of New Braunfels, in Senate district 25) don't have a value for `district`, so this conditional just stops the **School District: ** prompt from getting written if there's no data to put in after it.

Reply:
Interesting! I will investigate and add district info where possible

Reply:
Both of the ones I've found have a school district name in the campus name field.  So I think it's either deliberate, because it's about the district being a member, or values being in the wrong columns.